.github/workflows/ci.yml

```diff
@@ -48,3 +55,31 @@ jobs:
       - name: Check TinyGo version number consistency
         run: |
           builder/docker/tinygo/sanity.sh
+
+  # only run if reference is a tag
+  release:
+    if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
given that the workflow only runs on PRs against main with tags that begin with `v` as described here
```
on:
  pull_request:
  push:
    branches:
      - main
    tags:
      - v*
```

Do we need this?

Reply:
We do. At the top level, it means "run on every pull request, commit on main and tag" whereas, the if block only runs the `release` job only if it is a tag, but will wait for tests to pass based on the `need` block.